lib/adapters/class-vip-enterprise-search.php

```diff
@@ -30,6 +30,73 @@ public function action__ep_valid_response( $response ): void {
 		}
 	}
 
+	/**
+	 * Add aggs to formatted ES query args.
+	 *
+	 * @param array $formatted_args The formatted ES arg.
```

Comment:
```suggestion
	 * @param array $formatted_args Elasticsearch DSL.
```

Reply:
this is an array of formatted  ES args, it filters `ep_post_formatted_args`